templates/components/accordion.html.twig

```diff
@@ -0,0 +1,31 @@
+{% macro accordion(items, options) %}
+    {% set _id = options.id ?? ('accordion-' ~ random(0, 999)) %}
+    {% set _raw = options.raw ?? true %}
+    {% set _flush = options.flush ?? false %}
+    {% set _always_open = options.always_open ?? false %}
```

Comment:
As we said before @kevinpapst, we should keep the code to `CamelCase` only.
`always_open` ➡️ `alwaysOpen `

User macro's options should always be like that, but OUR code can be like `title_class` or `titleClass` ?